packages/xo-server-test/src/backupNg/backupNg.spec.js

```diff
@@ -12,14 +12,44 @@ const DEFAULT_SCHEDULE = {
   cron: '0 * * * * *',
 }
 
-const validateRootTask = (log, props) =>
-  expect(log).toMatchSnapshot({
+// it only support one schedule
+const validateBackupJob = async (xo, input, output) => {
```

Comment:
`input` and `output` names are very unclear.

Reply:
`jobInput` `jobOutput` will be more clearer?